src/metabase/util/tracking.clj

```diff
@@ -0,0 +1,54 @@
+(ns metabase.util.tracking
+  (:require [clj-http.client :as client]))
+
+(def ^:private ^:const ^String google-analytics-collect-url "http://www.google-analytics.com/collect")
+
+(def ^:private ^Integer anonymous-id
+  "Generate an anonymous id. Don't worry too much about hash collisions or localhost cases, etc.
+   The goal is to be able to get a rough sense for how many different hosts are throwing a specific error/event." 
+  (hash (str (java.net.InetAddress/getLocalHost))))
+
+(defn- google-analytics-tracking-code
+  "Get the GA id for the main metabase application"
+  []
+  (require 'metabase.public-settings)
+  ((resolve 'metabase.public-settings) :ga_code))
```

Comment:
e.g.

```clojure
 (resolve 'metabase.public-settings/ga-code)
;; -> #'metabase.public-settings/ga-code

(class (resolve 'metabase.public-settings/ga-code))
;; -> clojure.lang.Var

@ (resolve 'metabase.public-settings/ga-code)
;; -> "UA-60817802-1"

(class @ (resolve 'metabase.public-settings/ga-code))
;; -> java.lang.String
```

`#'symbol` (hash-quote) is just how Clojure represents vars.

`ga-code` is a symbol that names the `#'ga-code` var which holds the value `"UA-60817802-1"`. Does that make sense?